test/asn1numericstring_test.dart

```diff
@@ -0,0 +1,91 @@
+library asn1test;
```

Comment:
Thanks for including tests!

Reply:
No problem. I wouldn't know whether this works or not without it myself :)